### PR TITLE
MM-28941: stage progress bar

### PR DESCRIPTION
**Pull request by lieut-data**
#### Summary
One stage:
image

Two stages:
image

Three stages, 2nd stage active:
image

Ten stages, incident ended, themed:
image

Many stages, third stage active:
image

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28941

### CODE REVIEW

**Comment by lieut-data** [comment]
I didn't bother with an e2e test, mostly because it wasn't clear how to test purely visual content like this. For expediency in getting us to code freeze, I'm pushing as-is. Also, heads up @agarciamontoro that this builds off your pending changes.

File: webapp/src/components/rhs/incident_details.tsx [hunk 1]
```diff
@@ -137,7 +194,6 @@ const StyledButton = styled(BasicButton)<BasicButtonProps>`
         background: var(--button-bg);
         color: var(--button-color);
     `}
-}
```
Comment (agarciamontoro) [inline webapp/src/components/rhs/incident_details.tsx:140]: Oops

**Review by agarciamontoro — APPROVED** [review]
Nice!!! :tada:

**Comment by agarciamontoro** [comment]
Feel free to merge into my branch before #319 is merged!

**Pull request merged**